capsules/detector_safety_gear_openvino/backend.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict
+
+import numpy as np
+
+from vcap import (
+    DETECTION_NODE_TYPE,
+    OPTION_TYPE, BaseStreamState)
+from vcap_utils import BaseOpenVINOBackend
+
+
+class Backend(BaseOpenVINOBackend):
+    label_map: Dict[int, str] = {1: "unknown", 2: "safety vest",
+                                 3: "unknown",
+                                 4: "safety hat"}
+
+    def process_frame(self, frame: np.ndarray,
+                      detection_node: DETECTION_NODE_TYPE,
+                      options: Dict[str, OPTION_TYPE],
+                      state: BaseStreamState) -> DETECTION_NODE_TYPE:
+        input_dict, resize = self.prepare_inputs(frame)
+        prediction = self.send_to_batch(input_dict).get()
```

Comment:
```suggestion
        prediction = self.send_to_batch(input_dict).result()
```

Alex recently changed this to use futures instead of queues.